CONTRIBUTING.md

```diff
@@ -143,26 +143,28 @@ npm list -g oad # Should denote no packages installed
 - Save it to `openapi-platform` feed via [upstream feeds of `openapi-platform`].
+  - For the desired version, you must click `Save to feed` in the context menu of the hidden triple-dot button on the right. It will reveal on hover.
+  - You may need to wait a bit until the newest version shows up in `Upstream Versions` tab.
 - To make `openapi-alps` use the newly published package:
-  - Ensure that you update the minimum package version in  relevant `package.json` files and then run `rush update` and commit the changes.
-    - ❗ Do the update even if semver rules say the new package version is in scope. E.g. if the version is `~0.10.7` but you published `0.10.8`, then still update the entry to `~0.10.8`.
-    - If you will not modify any `package.json` at all, rush will not recognize the newly published package version. This can be worked around by running  `rush update --full`, but this is discouraged.
-    - Example file: [`private/azure-swagger-validation/azureSwaggerValidation/package.json`][azureSwaggerValidation package.json]
-    - Example `openapi-alps` PR doing the package update: [#537791]
-  - Redeploy relevant `openapi-alps` parts to pick up the new package, even if you didn't had to commit any changes. Follow [this doc][openapi-alps doc].
+  - Ensure that you update the minimum package version in relevant [`package.json`][azureSwaggerValidation package.json] files and then run `rush update` and commit the changes.
+  - ❗ Do the update even if semver rules say the new package version is in scope. E.g. if the version is `~0.10.7` but you published `0.10.8`, then still update the entry to `~0.10.8`.
+  - If you will not modify any `package.json` at all, rush will not recognize the newly published package version. This can be worked around by running  `rush update --full`, but this is discouraged.
+  - Example file: [`private/azure-swagger-validation/azureSwaggerValidation/package.json`][azureSwaggerValidation package.json].
+  - Example `openapi-alps` PR doing the package update: [#537791].
+- Redeploy relevant `openapi-alps` parts to pick up the new package, even if you didn't had to commit any changes. Follow [this doc][openapi-alps doc].
 
-[`openapi-diff`]: https://github.com/Azure/openapi-diff
-[`dotnet restore`]: https://learn.microsoft.com/en-us/dotnet/core/tools/dotnet-restore
+[#537791]: https://devdiv.visualstudio.com/DevDiv/_git/openapi-alps/pullrequest/537791
+[`@azure/oad` versions]: https://www.npmjs.com/package/@azure/oad?activeTab=versions
 [`dotnet build`]: https://learn.microsoft.com/en-us/dotnet/core/tools/dotnet-build
+[`dotnet restore`]: https://learn.microsoft.com/en-us/dotnet/core/tools/dotnet-restore
 [`dotnet test`]: https://learn.microsoft.com/en-us/dotnet/core/tools/dotnet-test
 [`npm link`]: https://docs.npmjs.com/cli/v10/commands/npm-link
-[the cache]: https://docs.npmjs.com/cli/v10/configuring-npm/folders#cache
 [`openapi-diff-build`]: https://dev.azure.com/azure-sdk/internal/_build/index?definitionId=7030
```

Comment:
@mikeharder @scbedd these two:

[`openapi-diff-build`]: https://dev.azure.com/azure-sdk/internal/_build/index?definitionId=7030
[`public.openapi-diff`]: https://dev.azure.com/azure-sdk/public/_build?definitionId=135&_a=summary

Now appear to be based on the same build definition. Is there any reason to keep the public one around or should I delete it?

Reply:
IMO keep both use `internal` for release and nightly (if you care about that for this project) and the `public` build for your CI checks.